content/en/docs/concepts/configuration/secret.md

```diff
@@ -24,6 +24,13 @@ a password, a token, or a key. Such information might otherwise be put in a
 Pod specification or in an image. Users can create Secrets and the system
 also creates some Secrets.
 
+Kubernetes Secrets are, by default, stored as unencrypted Base64 strings. Once written
+to Kubernetes, they can be retrieved - as plain text - by anyone with API access. In order
+to safely deploy Secrets, we recommend you (at a minimum):
+
+1. [Turn on Encryption at Rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/)
+2. [Enable RBAC rules that restrict reading and writing the secret](https://kubernetes.io/docs/reference/access-authn-authz/authorization/)
```

Comment:
```suggestion
2. [Enable RBAC rules that restrict reading and writing the Secret](https://kubernetes.io/docs/reference/access-authn-authz/authorization/)
```

Reply:
```suggestion
2. [Enable RBAC rules that restrict reading and writing the Secret](/docs/reference/access-authn-authz/authorization/)
```
(relative URL)
Does also assume the reader is using RBAC - a fairly safe assumption though